frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -394,6 +394,7 @@ function getMeshItems(
   maybeUnmappedSegmentId: number | null | undefined,
   visibleSegmentationLayer: APIDataLayer | null | undefined,
   voxelSizeFactor: Vector3,
+  meshFileMapping: string | null | undefined,
```

Comment:
IMO `meshFileMappingName` would be more accurate here as a name as the parameter value itself is the mapping name: `currentMeshFile?.mappingName`